### PR TITLE
style(ndm): add description to Hash method in util.go

### PR DESCRIPTION
**Pull request by rafa-acioly**
Fixes : #107

### CODE REVIEW

File: pkg/util/util.go [hunk 1]
```diff
@@ -105,7 +105,7 @@ func StrToInt32(val string) *int32 {
 	return n32
 }
 
-//get the hash of the string
+// Hash retrieve a encoded string using the md5 encrypt
```
Comment (sagarkrsd) [inline pkg/util/util.go]: Thanks @rafa-acioly .
Please replace retrieve -> retrieves and a -> an .

Comment (rafa-acioly) [reply pkg/util/util.go]: Sorry about that! 😄

**Comment by rafa-acioly** [comment]
@shovanmaity done.

**Comment by rafa-acioly** [comment]
@sagarkrsd done again! 👍

**Comment by Akash4927** [comment]
@rafa-acioly Can you please provide the link of the issue?

**Pull request merged**